wsagent/agent/src/main/resources/org.eclipse.che.ws-agent.script.sh

```diff
@@ -55,8 +55,12 @@ MACHINE_TYPE=$(uname -m)
 
 mkdir -p ${CHE_DIR}
 ${SUDO} mkdir -p /projects
-${SUDO} sh -c "chown -R $(id -u -n) /projects"
-${SUDO} chmod 755 /projects
+if [ "${CHANGE_PROJECTS_PERMISSIONS}" = false ]; then
+    :
```

Comment:
it's looking strange to have a no-io operation and the code in the else block

shouldn't it better to change the condition so the logic is in the "if" block ?